jest/setup.js

```diff
@@ -1,29 +1,13 @@
-import fs from 'fs';
-import path from 'path';
 import 'react-native-gesture-handler/jestSetup';
-import _ from 'underscore';
+import * as reanimatedJestUtils from 'react-native-reanimated/lib/reanimated2/jestUtils';
+import setupMockImages from './setupMockImages';
 
-require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests();
+setupMockImages();
+reanimatedJestUtils.setUpTests();
 
-jest.mock('react-native-blob-util', () => ({}));
-
-// These two mocks are required as per setup instructions for react-navigation testing
+// This mock is required as per setup instructions for react-navigation testing
 // https://reactnavigation.org/docs/testing/#mocking-native-modules
 jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
-jest.mock('react-native-reanimated', () => {
-    const Reanimated = require('react-native-reanimated/mock');
-    Reanimated.default.call = () => {};
-    return Reanimated;
-});
-
-// The main app uses a NativeModule called BootSplash to show/hide a splash screen. Since we can't use this in the node environment
-// where tests run we simulate a behavior where the splash screen is always hidden (similar to web which has no splash screen at all).
-jest.mock('../src/libs/BootSplash', () => ({
-    hide: jest.fn(),
-    getVisibilityStatus: jest.fn().mockResolvedValue('hidden'),
-}));
```

Comment:
Do we no longer need it?

Reply:
tests seem to pass without it so guessing whatever we needed it for we don't need it anymore.

Reply:
Cool then!